examples/globe.js

```diff
@@ -3,26 +3,52 @@
 
 // Define initial camera position
 var positionOnGlobe = { longitude: 2.351323, latitude: 48.856712, altitude: 25000000 };
+var promises = [];
 
 // `viewerDiv` will contain iTowns' rendering area (`<canvas>`)
 var viewerDiv = document.getElementById('viewerDiv');
+var miniDiv = document.getElementById('miniDiv');
 
 // Instanciate iTowns GlobeView*
 var globeView = new itowns.GlobeView(viewerDiv, positionOnGlobe, { renderer: renderer });
+var miniView = new itowns.GlobeView(miniDiv, positionOnGlobe, {
```

Comment:
Cound you add comments to your changes so this file can serve as documentation/tutorial?